### ISSUE TITLE
🐛 [Jellyfin] From some weeks ago Jellyfin can't start

### ISSUE THREAD

**Issue by txitxo0** [alexbelgium/hassio-addons#982]
### Description

From the last update (or one of the last) I am not able to start the Jellyfin addon, it starts but the webui doesn't work``

Using versions:
Home Assistant 2023.8.4
Supervisor 2023.08.3
Operating System 10.5
Jellyfin-addon: 10.8.10-6

### Reproduction steps

```bash
1. Having the default configuration in the addon (just using the local disk property pointing to my disck. //media/USB as always)
2. Click on start the addon
3. Go to the logs
4. See an error refering about the default jellyfin port 8096 is already in use. I tried changing the port but the same happened, I guess it is something from the internal network in the docker.
5. Going into ssh 
   sudo netstat -tulpn| grep 8097 (as the new webui port fordwarding from the addon config) no one is listening
```

### Addon Logs

```bash
[15:33:27] [ERR] [1] Main: Kestrel failed to start! This is most likely due to an invalid address or port bind - correct your bind configuration in network.xml and try again.
[15:33:27] [FTL] [1] Main: Error while starting server.
System.IO.IOException: Failed to bind to address http://0.0.0.0:8096: address already in use.
 ---> Microsoft.AspNetCore.Connections.AddressInUseException: Address already in use
 ---> System.Net.Sockets.SocketException (98): Address already in use
   at System.Net.Sockets.Socket.UpdateStatusAfterSocketErrorAndThrowException(SocketError error, String callerName)
   at System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.Sockets.Socket.Bind(EndPoint localEP)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportOptions.CreateDefaultBoundListenSocket(EndPoint endpoint)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportFactory.BindAsync(EndPoint endpoint, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure.TransportManager.BindAsync(EndPoint endPoint, ConnectionDelegate connectionDelegate, EndpointConfig endpointConfig, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.<>c__DisplayClass30_0`1.<<StartAsync>g__OnBind|0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.ListenOptions.BindAsync(AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.EndpointsStrategy.BindAsync(AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindAsync(IEnumerable`1 listenOptions, AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.BindAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.StartAsync[TContext](IHttpApplication`1 application, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Hosting.WebHost.StartAsync(CancellationToken cancellationToken)
   at Jellyfin.Server.Program.StartApp(StartupOptions options)
```

### Architecture

aarch64

### OS

HAos

**Comment by alexbelgium** [comment]
Hi `Failed to bind to address http://0.0.0.0:8096: address already in use.` means that you have already installed an addon using this port (Emby) ?

As they are using host network you can't have both running on the same system

**Comment by txitxo0** [comment]
Yep, I saw it and uninstall emby, I don't have anyother addon running nothing in 8096, but anyway that's why I was trying with another powrt for the ui, 8097.
![image](https://github.com/alexbelgium/hassio-addons/assets/12279324/f2a49173-6943-4d30-be72-cc7142a61fa1)

**Comment by alexbelgium** [comment]
Hi changing port in the ui doesn't work as it is using the network host so without port mapping. I don't know how we could see which add-on uses this port apart from stopping them all...

**Comment by txitxo0** [comment]
> Hi changing port in the ui doesn't work as it is using the network host so without port mapping. I don't know how we could see which add-on uses this port apart from stopping them all...

Hi!
Got it, okay so I will try stopping all add-ons and let's see what is happening. But it is so weird, I am 99% sure I don't have other add-ons using the same port. 

I'll you know, thank you for your quick response, asap as I can stop them all and try I'll post back here

**Comment by txitxo0** [comment]
Hi there, we both were totally rigth :)
The problem is not directly related with the addon. And I don't have more addon using the same ports.

I have turned off completely the host, boot again with all addons stopped. Starting ssh one and checking if the 8096 port is actually used with
sudo netstat -tulpn
It is not, finally!
Then I started and checked logs on every addon one by one 
And voila, jellyfin works now... Why it didn't before? I don't know, I guess some weird state...


Many thanks for your answers
